### PR TITLE
Hlee/intl stud card

### PR DESCRIPTION
**Pull request by fanglinfang**

### CODE REVIEW

File: myuw/templates/handlebars/card/international_student.html [hunk 1]
```diff
@@ -4,8 +4,71 @@
     <h3 class="myuw-card-title">
         International Student
     </h3>
+    <div class="links-container">
+        <h4 class="myuw-card-heading">Visa and Immigration Rules</h4>
 
-    <div class="myuw-card-section last">
+        <ul class="unstyled-list">
+        {{#if seattle}}
+	    <li><a href="https://iss.washington.edu/travel-visas/" target="_blank" data-linklabel="Travel and visas for Int'l students">Travel and Visas</a></li>
```
Comment (fanglinfang) [inline myuw/templates/handlebars/card/international_student.html]: Hi Hyunju, why these links are using https?  Could you check if just http works?

Comment (hlee2) [reply myuw/templates/handlebars/card/international_student.html]: I copy & paste the URL from ISS and CIE site. 
http reroutes to https.
Many of UW dept sites are 'https://....' nowadays and that trend will continue.

File: myuw/templates/handlebars/card/international_student.html [hunk 2]
```diff
@@ -4,8 +4,71 @@
...
+	    <li>
+		{{#if is_j1}}
+		    <a href="https://iss.washington.edu/regulations/j1/" target="_blank" data-linklabel="Visa and Immigration Rules">Visa and Immigration Rules for J-1 students</a>
+		{{else}}
+		    <a href="https://iss.washington.edu/regulations/f1/" target="_blank" data-linklabel="Visa and Immigration Rules">Visa and Immigration Rules for F-1 students</a>
+		{{/if}}
+	    </li>
+	    <li><a href="https://iss.washington.edu/new-students/faq/" target="_blank" data-linklabel="ISS: Frequently Asked Questions (FAQ)">Frequently Asked Questions (FAQ)</a></li>
+	    <li><a href="https://iss.washington.edu/employment/" target="_blank" data-linklabel="Int'l student employment">Employment and Internships</a></li>
```
Comment (klpittman) [inline myuw/templates/handlebars/card/international_student.html]: title case data-linklabel text.

File: myuw/templates/handlebars/card/international_student.html [hunk 3]
```diff
@@ -4,8 +4,71 @@
...
+	    <li><a href="https://www.trumba.com/calendars/sea_enroll-iss" target="_blank" data-linklabel="Deadlines and Workshops Calendar">Deadlines and Workshops Calendar</a></li>
+	{{/if}}
+
+	{{#if bothell}}
+	    <li><a href="http://www.uwb.edu/cie/current-students/travel" target="_blank" data-linklabel="Travel and visas for Int'l students">Travel and Visas</a></li>
```
Comment (klpittman) [inline myuw/templates/handlebars/card/international_student.html]: title case data-linklabel text.

File: myuw/templates/handlebars/card/international_student.html [hunk 4]
```diff
@@ -4,8 +4,71 @@
...
+	    <li><a href="https://www.uwb.edu/cie/current-students/immigration-documents" target="_blank" data-linklabel="Immigration Rules for F-1 students">Immigration Document for F-1 Students </a></li>
```
Comment (klpittman) [inline myuw/templates/handlebars/card/international_student.html]: title case data-linklabel text.

File: myuw/templates/handlebars/card/international_student.html [hunk 5]
```diff
@@ -4,8 +4,71 @@
...
+	    <li><a href="https://www.uwb.edu/cie/current-students/vacation-quarter" target="_blank" data-linklabel="Vacation Quarter">Vacation Quarter</a></li>
+	    <li><a href="http://www.uwb.edu/cie/employment" target="_blank" data-linklabel="Int'l student employment">Employment and Internships</a></li>
```
Comment (klpittman) [inline myuw/templates/handlebars/card/international_student.html]: title case data-linklabel text.

File: myuw/templates/handlebars/card/international_student.html [hunk 6]
```diff
@@ -4,8 +4,71 @@
...
+	    <li><a href="https://www.uwb.edu/cie/events" target="_blank" data-linklabel="Center for International Education Event Calendar">Center for International Education (CIE) Calendar</a></li>
+	{{/if}}
+
+	{{#if tacoma}}
+	    <li><a href="http://www.tacoma.uw.edu/iss/travel-visas" target="_blank" data-linklabel="Travel and visas for Int'l students">Travel and Visas</a></li>
```
Comment (klpittman) [inline myuw/templates/handlebars/card/international_student.html]: title case data-linklabel text.

File: myuw/templates/handlebars/card/international_student.html [hunk 7]
```diff
@@ -4,8 +4,71 @@
...
+	    <li><a href="http://www.tacoma.uw.edu/iss/maintaining-f-1-status" target="_blank" data-linklabel="Rules to maintain F-1 status">Rules to maintain F-1 status</a></li>
```
Comment (klpittman) [inline myuw/templates/handlebars/card/international_student.html]: title case data-linklabel text and the regular link text.

File: myuw/templates/handlebars/card/international_student.html [hunk 8]
```diff
@@ -4,8 +4,71 @@
...
+	    <li><a href="http://www.tacoma.uw.edu/isss/employment" target="_blank" data-linklabel="International Student Employment">International Student Employment</a></li>
+	    <li><a href="http://www.tacoma.uw.edu/international-student-services/events" target="_blank" data-linklabel="International Student and Scholar Services (ISSS) Calendar - Deadlines and Info Sessions">International Student and Scholar Services (ISSS) Calendar</a></li>
```
Comment (klpittman) [inline myuw/templates/handlebars/card/international_student.html]: This one is very long. It truncates in the Recently Visited at "International Student and Scholar..."

You might want to invert these so the differentiator portion is at the beginning: "ISSS Calendar (International Student and Scholar Services)" or "Calendar: ISSS"

Comment (klpittman) [reply myuw/templates/handlebars/card/international_student.html]: myuw__home

File: myuw/templates/handlebars/card/international_student.html [hunk 9]
```diff
@@ -4,8 +4,71 @@
...
+        <h4 class="myuw-card-heading">Academic Support</h4>
+        <ul class="unstyled-list">
+	{{#if seattle}}
+	    <li><a href="http://depts.washington.edu/aspuw/clue/home/" target="_blank" data-linklabel="Center for Learning and Undergraduate Enrichment (CLUE)">Center for Learning and Undergraduate Enrichment (CLUE)</a></li>
+	    <li><a href="http://depts.washington.edu/owrc" target="_blank" data-linklabel="Odegaard Writing and Research Center">Odegaard Writing and Research Center </a></li>
+	    <li><a href="http://aep.washington.edu/" target="_blank" data-linklabel="Academic English Program">Academic English Program</a></li>
+	{{/if}}
+
+	{{#if bothell}}
+	    <li><a href="http://www.uwb.edu/cie/current-students/writing-support" target="_blank" data-linklabel="Writing support and resources">Writing Support and Resources</a></li>
```
Comment (klpittman) [inline myuw/templates/handlebars/card/international_student.html]: title case

File: myuw/templates/handlebars/card/international_student.html [hunk 10]
```diff
@@ -4,8 +4,71 @@
...
+	    <li><a href="http://www.uwb.edu/admissions/apply/engprof" target="_blank" data-linklabel="English language proficiency">English Language Proficiency</a></li>
```
Comment (klpittman) [inline myuw/templates/handlebars/card/international_student.html]: title case

**Comment by klpittman** [comment]
I've added a note about the data-linklabel attribute in the Quick Links spec here: https://docs.google.com/document/d/1YgcK19K4Lod4eVD6VXEzs8VW0gmgwlhpGU_5Uk4dw-o/edit#heading=h.74z40bs8z11c

File: myuw/templates/handlebars/card/international_student.html [hunk 11]
```diff
@@ -4,8 +4,71 @@
...
+	    <li><a href="http://www.tacoma.uw.edu/international-student-services/events" target="_blank" data-linklabel="International Student and Scholar Services (ISSS) Calendar - Deadlines and Info Sessions">International Student and Scholar Services (ISSS) Calendar</a></li>
+	{{/if}}
+        </ul>
+    </div>
+
+    <div class="links-container">
+        <h4 class="myuw-card-heading">Academic Support</h4>
+        <ul class="unstyled-list">
+	{{#if seattle}}
+	    <li><a href="http://depts.washington.edu/aspuw/clue/home/" target="_blank" data-linklabel="Center for Learning and Undergraduate Enrichment (CLUE)">Center for Learning and Undergraduate Enrichment (CLUE)</a></li>
```
[the remark this answers is not in the document]
Comment (hlee2) [reply myuw/templates/handlebars/card/international_student.html]: I think the truncated link is better than 'CLUE'

File: myuw/templates/handlebars/card/international_student.html (hunk 8 above)
Comment (hlee2) [reply myuw/templates/handlebars/card/international_student.html]: change it to ISSS: Int'l...

File: myuw/templates/handlebars/card/international_student.html [hunk 12]
```diff
@@ -4,8 +4,71 @@
...
+	    <li><a href="http://www.uwb.edu/cie/employment" target="_blank" data-linklabel="Int'l Student Employment">Employment and Internships</a></li>
+	    <li><a href="https://www.uwb.edu/cie/events" target="_blank" data-linklabel="Center for International Education Event Calendar">Center for International Education (CIE) Calendar</a></li>
+	{{/if}}
+
+	{{#if tacoma}}
+	    <li><a href="http://www.tacoma.uw.edu/iss/travel-visas" target="_blank" data-linklabel="Travel and Visas for Int'l Students">Travel and Visas</a></li>
+	    <li><a href="http://www.tacoma.uw.edu/iss/maintaining-f-1-status" target="_blank" data-linklabel="Rules to Maintain F-1 Status">Rules to maintain F-1 status</a></li>
+	    <li><a href="http://www.tacoma.uw.edu/isss/employment" target="_blank" data-linklabel="International Student Employment">International Student Employment</a></li>
+	    <li><a href="http://www.tacoma.uw.edu/international-student-services/events" target="_blank" data-linklabel="ISSS Calendar (International Student and Scholar Services) ">International Student and Scholar Services (ISSS) Calendar</a></li>
```
Comment (klpittman) [inline myuw/templates/handlebars/card/international_student.html]: You've got an extra space at the end of the data-linklabel.

`data-linklabel="ISSS Calendar (International Student and Scholar Services) ">`

Comment (hlee2) [reply myuw/templates/handlebars/card/international_student.html]: fixed

**Pull request merged**